srcCpp/PerftestTransport.cxx

```diff
@@ -515,9 +515,19 @@ bool configureTransport(
 /******************************************************************************/
 /* CLASS CONSTRUCTOR AND DESTRUCTOR */
 
-PerftestTransport::PerftestTransport()
-        : dataLen(100)
+PerftestTransport::PerftestTransport() :
+        dataLen(100),
+        useMulticast(false),
+        throughputMulticastAddr("239.255.1.1"),
+        latencyMulticastAddr("239.255.1.2"),
+        announcementMulticastAddr("239.255.1.100"),
+        _LatencyTopicName("Latency"),
+        _AnnouncementTopicName("Announcement"),
+        _ThroughputTopicName("Throughput")
```

Comment:
It is possible to have the same name convention for all the member of the class:
latencyTopicName
announcementTopicName
throughputTopicName

Reply:
I suggested to remove these variables, we came up with an idea.

Reply:
DONE